### ISSUE TITLE
Add tabs for the thread list

### ISSUE THREAD

**Issue by beardog108** [beardog108/MicroTXT#5]
From @eArabic originally on issue #4 

The thread list needs tabs

**Comment by beardog108** [comment]
@eArabic, for this feature I am using sqlite3. I know one of the points of this project is to have no database, but a tabbed thread system is complicated and would be messy and slow without a database system. 

I would basically just be reinventing sqlite3 anyway by making it without, so unless you have any serious objections, this is how I have been approaching this feature. 

It is not done yet, but it will be in a relatively short amount of time.

**Comment by eArabic** [comment]
Thank you very much, I wish I knew programming be offered something, but I new to programming.
I hope that stays fast, without a database, and work from anywhere in the server, without any settings.
The best feature are now his speed and his work from anywhere and without any settings or database.
His idea of a beautiful, wonderful, make it easier to communicate with others.
thank you very much

**Comment by beardog108** [comment]
Do you not like databases because they are difficult to install? Sqlite3 is considerably easier to use than MySQL. 

You won't need to add any settings, but you may need to install Sqlite3 if it is not already. As I said this is the most logical way for me to implement the tab system.

However, If you are operating in a server where Sqlite3 cannot be used at all, I could perhaps try to use json as a fallback.

**Comment by beardog108** [comment]
I added basic next & back buttons for when too many threads are showing. It also lists the author name too, but remember there is no authentication for authors besides the tripcode system which should be manually checked.

**Comment by beardog108** [comment]
The new database system isn't compatible with previous versions by the way. I should bump the version to 1.0 to reflect that.

**Comment by beardog108** [comment]
I'll try to make a fallback for systems without sqlite, which will function like it has in the past, with no next/back buttons.

**Comment by eArabic** [comment]
Thank you very much for the update, I will test him, very pleased update.
If it is the process of adding search all posts.
If it is activated Find all posts by a writer by clicking on his name and the activation link.
The search for a word in the site if necessary is added search feature.
Is the need to control panel director controls to posts (delete, modify, or close the Post, and other features ...)

Thank you so much wonderful work and beautiful.

**Comment by eArabic** [comment]
Hello
I noticed the order of older posts at the top and the newest at the bottom
How is the sort of new articles to be at the top.

**Comment by beardog108** [comment]
Yep, I agree with everything you said. It will take some more time but I will add these things.

**Comment by beardog108** [comment]
Thanks for your patience. I will try to get reverse order done soon.

**Comment by beardog108** [comment]
I almost have the older at top issue fixed.

However, something I wanted to mention was when you said "If it is activated Find all posts by a writer by clicking on his name and the activation link", I could perhaps do this with the tripcode feature, however as I'm sure you know there is no login as MicroTXT is a textboard, which like imageboards, do not use registration systems.

I'll consider doing this with tripcodes :)

**Comment by beardog108** [comment]
Ok I added showing newest posts first, but I'm not confident that it is stable, so I added it to a new branch: https://github.com/beardog108/MicroTXT/tree/reverse-order

**Comment by beardog108** [comment]
Just to let you know, I patched a vulnerability with the tripcode system in the master branch: https://github.com/beardog108/MicroTXT/commit/cd899a1333b9d069d2bc826573cd66e37cc7ca2f

Salts were not being applied to the tripcode because of a scope issue, therefore any preexisting tripcodes are potentially weak and easily crackable if they used a bad password.

**Comment by eArabic** [comment]
Thank you very much I will experiment and test, thank you very much to communicate and update.
